src/front/spv/mod.rs

```diff
@@ -1731,29 +1738,23 @@ impl<I: Iterator<Item = u32>> Parser<I> {
                             self.lookup_expression.lookup(arg_id)?.handle
                         };
 
-                        let constant_handle = const_arena.fetch_or_append(crate::Constant {
-                            name: None,
-                            specialization: None,
-                            inner: crate::ConstantInner::Scalar {
-                                width: 4,
-                                value: crate::ScalarValue::Float(match gl_op {
-                                    Glo::Radians => std::f64::consts::PI / 180.0,
-                                    Glo::Degrees => 180.0 / std::f64::consts::PI,
-                                    _ => unreachable!(),
-                                }),
-                            },
-                        });
-
-                        let expr_handle =
-                            expressions.append(crate::Expression::Constant(constant_handle));
+                        let constant = match gl_op {
+                            Glo::Radians => {
+                                self.maths_constant_expressions[DEGREES_TO_RADIANS_CONST_INDEX]
```

Comment:
Let's freeze this until #817 lands (and then re-implement on top of it). After it, you'd just call `self.get_constant_scalar(..)` here, and you could remove `maths_constant_expressions` entirely

Reply:
this is merged now

Reply:
I can only see a `get_constant_scalar` function in `back/spv`..

Reply:
oh, sorry, I confused myself